internal/logstore/emitters/execution/record.go

```diff
@@ -0,0 +1,26 @@
+package execution
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/zitadel/zitadel/internal/logstore"
+)
+
+var _ logstore.LogRecord = (*Record)(nil)
+
+type Record struct {
+	Timestamp  time.Time
```

Comment:
as in access

Reply:
What do you mean exactly? It's a timestamp, not a duration, here.